controllers/ssp_controller.go

```diff
@@ -94,14 +94,11 @@ func NewSspReconciler(client client.Client, uncachedReader client.Reader, infras
 
 var _ reconcile.Reconciler = &sspReconciler{}
 
-// +kubebuilder:rbac:groups=ssp.kubevirt.io,resources=ssps,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=ssp.kubevirt.io,resources=ssps/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=ssp.kubevirt.io,resources=ssps,verbs=list;watch;update
```

Comment:
Why does SSP need to update the SSP CR?

Reply:
Without that verb, the BeforeSuite fails on this assert https://github.com/kubevirt/ssp-operator/blob/25b2b9a704d9973fbfa39f6b97eca64d5fffb7ec/tests/tests_suite_test.go#L514. So I suppose that SSP needs to update at least the `Generation` field in presence changes to it.

Reply:
Ahh nice! Thanks for the clarifications :-)